Terminal.Gui/Drawing/ColorParseException.cs

```diff
@@ -3,90 +3,87 @@
+        Reason = reason;
+    }
 
-	/// <summary>
-	///   Creates a new instance of a <see cref="ColorParseException" /> populated with the provided values.
-	/// </summary>
-	/// <param name="colorString">The text that caused this exception, as a <see langword="string" />.</param>
-	/// <param name="badValue">
-	///   The specific value in <paramref name="colorString" /> that caused this exception.
-	/// </param>
-	/// <param name="badValueName">
-	///   The name of the value (red, green, blue, alpha) that <paramref name="badValue" /> represents.
-	/// </param>
-	/// <param name="reason">The reason that <paramref name="badValue" /> failed to parse.</param>
-	internal ColorParseException (in ReadOnlySpan<char> colorString, string reason, in ReadOnlySpan<char> badValue = default, in ReadOnlySpan<char> badValueName = default) : base (DefaultMessage)
-	{
-		ColorString = colorString.ToString ();
-		BadValue = badValue.ToString ();
-		BadValueName = badValueName.ToString ();
-		Reason = reason;
-	}
+    /// <summary>Creates a new instance of a <see cref="ColorParseException"/> populated with the provided values.</summary>
+    /// <param name="colorString">The text that caused this exception, as a <see langword="string"/>.</param>
+    /// <param name="badValue">The specific value in <paramref name="colorString"/> that caused this exception.</param>
+    /// <param name="badValueName">The name of the value (red, green, blue, alpha) that <paramref name="badValue"/> represents.</param>
+    /// <param name="reason">The reason that <paramref name="badValue"/> failed to parse.</param>
+    internal ColorParseException (
+        in ReadOnlySpan<char> colorString,
+        string? badValue = null,
+        string? badValueName = null,
+        string? reason = null
+    ) : base (DefaultMessage) {
+        ColorString = colorString.ToString ();
+        BadValue = badValue;
+        BadValueName = badValueName;
+        Reason = reason;
+    }
 
-	/// <summary>
-	///   Creates a new instance of a <see cref="ColorParseException" /> populated with the provided values.
-	/// </summary>
-	/// <param name="colorString">The text that caused this exception, as a <see langword="string" />.</param>
-	/// <param name="badValue">
-	///   The specific value in <paramref name="colorString" /> that caused this exception.
-	/// </param>
-	/// <param name="badValueName">
-	///   The name of the value (red, green, blue, alpha) that <paramref name="badValue" /> represents.
-	/// </param>
-	/// <param name="reason">The reason that <paramref name="badValue" /> failed to parse.</param>
-	internal ColorParseException (in ReadOnlySpan<char> colorString, string? badValue = null, string? badValueName = null, string? reason = null) : base (DefaultMessage)
-	{
-		ColorString = colorString.ToString ();
-		BadValue = badValue;
-		BadValueName = badValueName;
-		Reason = reason;
-	}
+    internal const string DefaultMessage = "Failed to parse text as Color.";
```

Comment:
(out of scope)

Should I consider not using a const here, and making this localizable?